### PR TITLE
Created percentile function in the numpy frontend

### PR DESCRIPTION
**Pull request by Tiago-B-C-Reis**
Add percentile to function frontend and respective unitest + start of percentile def on functional Ivy.
percentile #8952

### CODE REVIEW

File: ivy/functional/frontends/numpy/statistics/percentile.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# global
```
Comment (jkeane508) [inline ivy/functional/frontends/numpy/statistics/percentile.py:1]: Here we don't need a new file for percentile, we should implement this in order_statistics.py

File: ivy/functional/ivy/statistical.py [hunk 2]
```diff
@@ -1260,3 +1260,17 @@ def einsum(
 
     """
     return current_backend(operands[0]).einsum(equation, *operands, out=out)
+
```
Comment (jkeane508) [inline ivy/functional/ivy/statistical.py:1263]: This is not part of the task, and of currently we are not adding any more functions to the Ivy API

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_percentile.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
+# global
```
Comment (jkeane508) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_percentile.py:1]: This should be in test_order_statistics.py not in its own file.

**Review by jkeane508 — COMMENTED** [review]
Hi, thanks for submitting this PR some changes are needed :)

I would consider reading over the docs to better understand the structure of the project. Moreover, you seem to have lint errors in your implementations, you can add a pre-commit hook to prevent these errors - https://lets-unify.ai/ivy/contributing/setting_up.html#pre-commit

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_percentile.py (hunk 3 above)
Comment (Tiago-B-C-Reis) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_percentile.py:1]: Hello, thank you very much for the feedback, I will start working on this but I just have one question:
Should this 'percentile' task be attached to this issue #8953, like the median for example?

image

**Pull request closed without merging**